elk_6x_integration/README.md

```diff
@@ -0,0 +1,92 @@
+# Integrating rpc-ceph with elk_metrics_6x
+
+You can collect metrics from Ceph nodes using [elk_metrics_6x](https://github.com/openstack/openstack-ansible-ops/tree/master/elk_metrics_6x) via metricbeat.
+Metricbeat will be used to collect host metrics from the Ceph nodes and the prometheus plugin for metricbeat
+will be used to collect Ceph metrics from the Prometheus plugin for the Ceph Mgr service.
+
+
+## Install Metricbeats on Ceph Nodes
+
+### Add Ceph Nodes to OSA Inventory
```

Comment:
I'm not sure having the ceph nodes in the OSA inventory is needed in production. My worry is that OSA will assume that it needs to set up ceph-mons containers if this ceph.yml file is added before Openstack installation.

Reply:
I will admit thought that it has been awhile since I last checked how OSA deals with ceph nodes in it's inventory.

Reply:
In the example I am using an unused group name ceph-ext as "ceph external nodes" to avoid any conflicts with existing OSA groups ceph-osd, ceph-mon, etc.  Are you thinking we just use a static inventory file outside of OSA and apply the Metricbeat role using it?  That should work fine and should also be safer.

Reply:
My goal is to keep Openstack and Ceph as separate as possible. That sounds good, and I have a feeling that things will be even easier/safer when we start deploying from the logging nodes.

Reply:
Sounds good.  I am going to leave this open until we get through the first deployment and based off of that experience I will update the doc and inventory to match.